play/src/components/FooterContent.tsx

```diff
@@ -32,15 +32,17 @@ const Avatar = styled('img')({
 interface HeaderProps {
   avatar: number;
   teamName: string;
-  newPoints: number;
-  score: number | null;
+  newPoints?: number;
+  score: number;
+  handleUpdateScore?: (newScore: number) => void;
```

Comment:
I'm using the `?` property for newPoints because the `FooterContent` shows up throughout the app but `newPoints` is only altered in `PHASE_1_RESULTS` and `PHASE_2_RESULTS.` So on all other screens, I don't pass a prop here for `newPoints.` Let me know if this seems weird to you though.